### DISCUSSION TITLE
cannot change resolution of GIF file, its always 72ppi

### DISCUSSION THREAD

**Discussion by jetpack1116** [ImageMagick/ImageMagick#5171] [Help] [answered]
Given original image 
--------------
resolution ; 300ppi
format: PNG
--------------

I want to convert it to GIF with resolution 144ppi

So I use the command:
`magick convert -units PixelsPerInch -density 144 testImage.png GIF:C:\test\image.gif`

But when I checked the GIF output, its resolution is 72ppi.

I reposition `-density` option but still same result.

``magick convert -units PixelsPerInch testImage.png -density 144 GIF:C:\test\image.gif``

This does not happen when I convert PNG to JPEG. 

I use ImageMagick-7.1.0-Q16-HDRI

Is this ImageMagick bug?

**Comment by snibgo** [comment]
As far as I know, GIF doesn't have a metadata field for image resolution.

**Reply by jetpack1116** [reply]
Hi @snibgo 

You mean that the GIF resolution cannot be changed because it has no `image resolution` metadata field?

**Comment by snibgo** [comment, marked as answer]
Images in GIF files have no resolution. Setting a resolution of an image (with "-density") has no effect on the GIF file.
